lib/widgets/color.dart

```diff
@@ -16,3 +16,25 @@ Color clampLchLightness(Color color, num lowerLimit, num upperLimit) {
     .copyWith(lightness: asLab.lightness.clamp(lowerLimit, upperLimit))
     .toColor();
 }
+
+extension ColorExtension on Color {
+  /// A 32 bit value representing this sRGB color.
+  ///
+  /// If [colorSpace] is not [ColorSpace.sRGB], do not use this.
+  ///
+  /// The bits are assigned as follows:
+  ///
+  /// * Bits 24-31 are the alpha value.
+  /// * Bits 16-23 are the red value.
+  /// * Bits 8-15 are the green value.
+  /// * Bits 0-7 are the blue value.
+  ///
+  /// This is the same form that [Color.new] takes.
+  int get argbInt {
+    assert(colorSpace == ColorSpace.sRGB);
+
+    // If the soon-to-be-deprecated [value] getter is removed, reimplement
+    // or migrate callers away from this form.
+    return value;
```

Comment:
Per the suggestion [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/Deprecation.20analyzer.20info/near/1936951), we probably just want to port that implementation here.